### PR TITLE
Tennessee

### PR DESCRIPTION
**Pull request by SteveWallace**
TN scraper added

### CODE REVIEW

File: scrapers.js [hunk 1]
```diff
@@ -913,34 +1019,95 @@ let scrapers = [
...
+  },
+  {
+    state: 'TN',
+    country: 'USA',
+    url: 'https://www.tn.gov/health/cedep/ncov.html',
+    scraper: async function() {
+      let counties = [];
+      let $ = await fetch.page(this.url);
+      let $table = $('th:contains("Case Count")').closest('table');
+      console.log($table);
```
Comment (osvyg) [inline scrapers.js]: remove log

Comment (SteveWallace) [reply scrapers.js]: derp

File: scrapers.js [hunk 2]
```diff
@@ -253,17 +256,18 @@ let scrapers = [
...
+          return row.data === latestDate;
+        })
+        .map(row => {
+          return {
+            recovered: parse.number(row.dimessi_guariti),
+            deaths: parse.number(row.deceduti),
+            cases: parse.number(row.totale_casi),
+            county: parse.string(row.denominazione_regione)
+          };
+        });
```
Comment (lazd) [inline scrapers.js]: Erm, I'm confused, why is Italy modified here, and why does the existing code not have `row.data.substr(0, 10)`? Can you rebase from master?

**Comment by SteveWallace** [comment]
@lazd  this ones ready to go

**Comment by lazd** [comment]
Hmm, something is funky with the history, I still see that `console.log` and the other weirdness. I can pull it in manually if getting the branches in shape is a pain.

**Pull request closed without merging**